models/edr/elementary_exposures/enriched_exposures.sql

```diff
@@ -16,4 +16,4 @@ elementary_exposures as (
 
 {# Union without duplicates where elementary_exposures has prio #}
 select * from dbt_exposures where not exists (select 1 from elementary_exposures where dbt_exposures.unique_id = elementary_exposures.unique_id)
-union select * from elementary_exposures
+union{%- if target.type == 'bigquery' -%} distinct {%- endif -%}select * from elementary_exposures
```

Comment:
```suggestion
union {%- if target.type == 'bigquery' -%}distinct {%- endif -%}select * from elementary_exposures
```
so it won't result with `unionselect * from elementary_exposures`

Reply:
Can you remove the `-` in general? They remove the whitespaces.